notebooks/official/matching_engine/sdk_matching_engine_create_text_to_image_embeddings.ipynb

```diff
@@ -129,7 +129,7 @@
     {
```

Comment:
<p>Google Cloud Vision -&gt; Cloud Vision</p><p> </p><p>See .. -&gt; Learn more about</p><p> </p><p>Should have heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>Done.</p>